README.md

```diff
@@ -37,7 +37,7 @@ In addition, this new version brings the syntax highlighting up to date with all
 ```json
 {
   "terraform.languageServer": {
-    "enabled": true,
+    "external": true,
```

Comment:
I am a bit unclear on the history here, but the code lead me to believe the old plugin had a setting called `enabled` but our plugin uses a new property called `external`? This README I think was a typo (not reflecting the new property)

Reply:
Yes, that's the case. It probably wasn't too well thought out on my side, but when we were originally prototyping we had both an internal language server option and terraform-ls to switch between. It does give us a flag to look for on old configs though.